lib/src/working_copy.rs

```diff
@@ -864,24 +864,25 @@ impl TreeState {
         repo_path: &RepoPath,
         disk_path: &Path,
         conflict_id: ConflictId,
-        file_type: FileType,
+        executable: bool,
     ) -> Result<TreeValue, SnapshotError> {
         let mut file = File::open(disk_path).unwrap();
         let mut content = vec![];
         file.read_to_end(&mut content).unwrap();
-        let conflict = self.store.read_conflict(&repo_path, &conflict_id)?;
+        let conflict = self.store.read_conflict(repo_path, &conflict_id)?;
         if let Some(new_conflict) = conflict
-            .update_from_content(self.store.as_ref(), &repo_path, &content)
+            .update_from_content(self.store.as_ref(), repo_path, &content)
```

Comment:
nit: run `clippy --fix` at the previous revision

Reply:
Done.